### DISCUSSION TITLE
4th iteration in Rust?

### DISCUSSION THREAD

**Discussion by VuiMuich** [aristocratos/bpytop#307] [Ideas]
Just for the meme sake when you are fine with `btop` a reimplementation in rust called `brtop` (or even `brrrtop`) would be hilarious 😆

**Comment by aristocratos** [comment]
@VuiMuich 
Noo, I'm slowly peeling off abstraction layers down to assembly. So the last iteration would be `asstop` and replace your BIOS 🔢
